### PR TITLE
fix(chat): Edit last message hotkey on macOS

### PR DESCRIPTION
**Pull request by ShGKme**
### ☑️ Resolves

* Follow-up: https://github.com/nextcloud/spreed/issues/13654
* <kbd>Ctrl+⬆️</kbd> is a system hotkey on Mac and doesn't work as web-page hotkey
  * Added <kbd>Cmd(Meta)+⬆️</kbd>
* Also fixed hotkeys in the help in the settings dialog

### 🖼️ Screenshots / Screencasts

🏚️ Before | 🏡 After
-- | --
image | image
image | image

### 🏁 Checklist

- [x] 🌏 Tested with different browsers / clients:
  - [x] Chromium (Chrome / Edge / Opera / Brave)
  - [x] Firefox
  - [x] Safari
  - [x] Talk Desktop
  - [ ] Not risky to browser differences / client
- [x] 🖌️ Design was reviewed, approved or inspired by the design team
- [x] ⛑️ Tests are included or not possible
- [x] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required

### CODE REVIEW

File: src/components/NewMessage/NewMessage.vue [hunk 1]
```diff
@@ -82,6 +82,7 @@
 					dir="auto"
 					@keydown.esc="handleInputEsc"
 					@keydown.ctrl.up="handleEditLastMessage"
+					@keydown.meta.up="handleEditLastMessage"
```
Comment (Antreesy) [inline src/components/NewMessage/NewMessage.vue:85]: meta is a 'Windows' key also. maybe replace it with `useHotKey()`?

File: src/utils/browserCheck.ts [hunk 2]
```diff
@@ -10,6 +10,13 @@ import { t } from '@nextcloud/l10n'
 
 const parser = new UAParser()
 const browser = parser.getBrowser()
+const os = parser.getOS()
+
+/**
+ * Per-OS flags
+ */
+
+export const isMac = os.name === 'Mac OS'
```
Comment (Antreesy) [inline src/utils/browserCheck.ts:19]: Shouldn't we update in the meantime?
https://docs.uaparser.dev/intro/whats-new.html#version-2-0

File: src/components/NewMessage/NewMessage.vue (hunk 1 above)
Comment (ShGKme) [reply src/components/NewMessage/NewMessage.vue:85]: > meta is a 'Windows' key also

It is not a problem, it is ignored on Windows and, afaik, all (most?) Linux DEs (Checked on GNOME, KDE, xfce, Cinnamon)

> maybe replace it with `useHotKey()`?

I'd avoid having a global listener here

File: src/utils/browserCheck.ts (hunk 2 above)
Comment (ShGKme) [reply src/utils/browserCheck.ts:19]: We can

**Review by DorraJaouad — APPROVED** [review]
Tested on Windows (at least it's confirmed that it remains working for Win :') )

File: src/utils/browserCheck.ts (hunk 2 above)
Comment (Antreesy) [reply src/utils/browserCheck.ts:19]: Then let's follow-up, if you don't want to mix PRs

**Pull request merged**